test/NVQPP/auto_kernel.cpp

```diff
@@ -6,8 +6,10 @@
  * the terms of the Apache License 2.0 which accompanies this distribution.    *
  ******************************************************************************/
 
-// RUN: nvq++ --enable-mlir -v %s -o %t && %t | FileCheck %s
 // RUN: nvq++ -std=c++17 --enable-mlir %s -o %t && %t | FileCheck %s
+// RUN: if [ $(echo %cpp_std | cut -c4- ) -ge 20 ]; then \
```

Comment:
What is this conditional about?

Reply:
> What is this conditional about?

See https://github.com/NVIDIA/cuda-quantum/pull/1101#discussion_r1457449084

Reply:
Thanks, Ben. I saw that after I started commenting.

I may be wrong, but this is running an entire new build so that it can stick `-std=c++17` in on a few more tests use `nvq++` and `cudaq-quake`?

Reply:
Yes, that's my interpretation...it's giving more test coverage to the `c++17` code than previously existed.